packages/devtools_app/lib/src/inspector/inspector_tree.dart

```diff
@@ -42,8 +43,8 @@ extension InspectorColorScheme on ColorScheme {
 const double iconPadding = 5.0;
 const double chartLineStrokeWidth = 1.0;
 const double columnWidth = 16.0;
-const double verticalPadding = 10.0;
-const double rowHeight = 24.0;
+double get verticalPadding => 10.0 * ideTheme.fontSizeFactor();
```

Comment:
For discussion: it might be a good idea to round the verticalPadding and row height o the nearest px to ensure the ux is always in whole number of pixels unless there is a strong reason not to.

Reply:
That sounds reasonable. I added a util function for this. Do you think it's necessary to restrict font sizes to reasonable min/max numbers?